src/components/inputManager.js

```diff
@@ -18,27 +17,18 @@ define(['playbackManager', 'focusManager', 'appRouter', 'dom'], function (playba
     }
 
     function select(sourceElement) {
-
         sourceElement.click();
     }
 
     var eventListenerCount = 0;
     function on(scope, fn) {
         eventListenerCount++;
-        dom.addEventListener(scope, 'command', fn, {
-
-        });
+        dom.addEventListener(scope, 'command', fn, {});
     }
 
     function off(scope, fn) {
-
-        if (eventListenerCount) {
-            eventListenerCount--;
-        }
-
-        dom.removeEventListener(scope, 'command', fn, {
-
-        });
+        eventListenerCount--;
```

Comment:
This could cause an error if eventListenerCount is undefined.

Reply:
I don't think it should since the variable is initialized right when it gets created, but I can add this back if you want. The increment section doesn't have a check.

Reply:
I would add it back just to err on the side of safety.